travisci/python-unittest_harness.sh

```diff
@@ -15,7 +15,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-PYTHON_SOURCE_LOCATIONS=('scripts')
+PYTHON_SOURCE_LOCATIONS=('src/python/tests')
```

Comment:
Python scripts that are under `scripts/` should also be tested (compilation only)